rskj-core/src/main/java/co/rsk/cli/tools/ImportState.java

```diff
@@ -47,8 +47,9 @@ public static void main(String[] args) {
 
     @Override
     protected void onExecute(@Nonnull String[] args, @Nonnull RskContext ctx) throws Exception {
-        String databaseDir = ctx.getRskSystemProperties().databaseDir();
-        KeyValueDataSource trieDB = LevelDbDataSource.makeDataSource(Paths.get(databaseDir, "unitrie"));
+        RskSystemProperties rskSystemProperties = ctx.getRskSystemProperties();
+        String databaseDir = rskSystemProperties.databaseDir();
+        KeyValueDataSource trieDB = KeyValueDataSource.makeDataSource(Paths.get(databaseDir, "unitrie"), rskSystemProperties.databaseKind());
```

Comment:
shouldn't we validate db kind in this place too, same as it's done in the `Start` class?

btw.. what about having that validation logic in one common place, eg. in the `KeyValueDataSource.makeDataSource()` method?